server/db/addUser.js

```diff
@@ -0,0 +1,18 @@
+var client = require('./init.js');
+
+function addUser (data, url) {
+
+    var userKey = 'user:' + data.profile.id;
+
+    client.hmsetAsync(userKey,
+                     'firstName', data.profile.name.first,
+                     'lastName', data.profile.name.last,
+                     'id', data.profile.id,
+                     'token', data.token,
+                     'photoURL', url)
+        .catch((error) => {
+            console.error('Error adding user to db:', error.cause);
+        });
+}
+
+module.exports = addUser;
```

Comment:
Why is this whole file showing up in green? I thought we had added this on friday? same goes for the one below. shouldn't the changes be highlighted instead of the whole file in green?